DigitalOffice/Services/CompanyService/Database/Entities/DbPosition.cs

```diff
@@ -8,7 +8,6 @@ public class DbPosition
     {
         [Key]
         public Guid Id { get; set; }
-        [Required]
```

Comment:
Why?

Reply:
Idk what was the purpose but now it's fixed.